routes/web.php

```diff
@@ -16,3 +17,11 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Auth::routes();
+
+// Route::view('/player_register', 'auth.player_register')->name('player_register');
+
+Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+Route::get('/choose_register', [App\Http\Controllers\Auth\RegisterController::class, 'choose'])->name('choose');
```

Comment:
```suggestion
Route::get('/choose_register', [App\Http\Controllers\Auth\RegisterController::class, 'choose'])->name('choose_register');
```

にしておいたほうがよさそうです。  `register` だけだと、なんの機能なのか判別がつきにくいのと、今後新しいルーティングを追加するときに名前が競合する可能性があるためですね！